src/config/QuoteList.ts

```diff
@@ -177,4 +177,16 @@ export const QuoteList: Quote[] = [
     text: "Develop success from failures. Discouragement and failure are two of the surest stepping stones to success.",
     author: "Dale Carnegie",
   },
+  {
+    text: "If you think you're gonna crash, accelerate more you idiot",
+    author: "Suga (BTS)",
+  },
+  {
+   text: "Someone else will surely be comforted by looking at your thorns, we are each other's night scenery, each other's moon",
+   author: "RM (BTS)",
```

Comment:
```suggestion
    text: "Someone else will surely be comforted by looking at your thorns, we are each other's night scenery, each other's moon",
    author: "RM (BTS)",
```
Need the proper spacing here.